docs/modules/traverse.md

```diff
@@ -0,0 +1,13 @@
+# traverse
+
+traverse has 66 total transitive dependencies, is CJS-only and is projected to get bigger over time.
```

Comment:
i wouldn't mention the exact number since it may change

```suggestion
traverse has many transitive dependencies, is CommonJS-only and is projected to get bigger over time.
```